workspace/__lib__/xod/core/clock/patch.xodp

```diff
@@ -37,10 +40,22 @@
       "id": "HJU8CE2lW",
       "label": "TICK",
       "position": {
-        "x": 138,
-        "y": 224
+        "x": 102,
+        "y": 204
       },
       "type": "xod/patch-nodes/output-pulse"
+    },
+    {
+      "boundValues": {
+        "__out__": true
+      },
+      "id": "r1jzp_DTZ",
+      "label": "EN",
```

Comment:
I forgot a description. Proposal:

“Is the clock enabled, i.e. produces ticks? At the moment when set to true, starts counting from scratch.”